README.md

```diff
@@ -2,3 +2,6 @@
 Catalogus voor de DataPunt APIs
 
 the webpage at api.data.amsterdam.nl
+
+# Local development
+Simulator available at http://localhost:8888/Simulator/index.html
```

Comment:
Tellus Webconnector is available on http://localhost:8080/webconnector/tellus/

Simulator is different repo. It's very nice to have it included, but the setup to start the simulator should probably also be described.

Reply:
Apparently there already was a description in the webconnector and webconnector-tellus folders, so a referenced those here instead